cygnus-common/pom.xml

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
Looking to https://github.com/telefonicaid/fiware-cygnus/pull/1896/files, CNR file should be flushed in the release closing PR.

`rm CHANGES_NEXT_RELEASE ; touch CHANGES_NEXT_RELEASE` does the trick :)

Reply:
fixed in be3d8a35bc206ccf4b1dcab2d3721bab6578ab80